rotkehlchen/accounting/structures/base.py

```diff
@@ -169,22 +169,22 @@ def deserialize_from_db(cls, entry: HISTORY_EVENT_DB_TUPLE_READ) -> 'HistoryBase
     @property
     def serialized_event_identifier(self) -> str:
         """Take a HistoryBaseEntry's event_identifier and returns a string representation."""
-        if self.location == Location.KRAKEN:
+        try:
             return self.event_identifier.decode()
-
-        hex_representation = self.event_identifier.hex()
-        if hex_representation.startswith('0x') is True:
-            return hex_representation
-        return '0x' + hex_representation
+        except UnicodeDecodeError:
+            hex_representation = self.event_identifier.hex()
+            if hex_representation.startswith('0x') is True:
+                return hex_representation
+            return '0x' + hex_representation
```

Comment:
This looks bad. You are relaying on a error to change the logic. This is bad. Why did you change this? If is an error that you found in a test you need to find why it happened and see how the logic should be, not relay on a error catch like this